src/mon/PGMap.cc

```diff
@@ -4550,8 +4556,41 @@ int reweight::by_utilization(
       osd_util.second = (double)p.second.kb_used / (double)p.second.kb;
     }
     util_by_osd.push_back(osd_util);
+
+    // Overweighted devices and their weights are factors to
```

Comment:
I was using both single line and multi-line comments. I changes everything to multi-line comments, to be consistent, as the Ceph coding style guide mentions.